Hymns/Display/HymnLyricsView.swift

```diff
@@ -3,7 +3,8 @@ import Resolver
 
 public struct HymnLyricsView: View {
 
-    @ObservedObject private var viewModel: HymnLyricsViewModel
+    //Pass in from DetailHymnScreen
```

Comment:
just FYI. In general, I don't think we need comments like this, since it effectively couples the hymn lyrics view with the detail hymn view which is what we are trying to avoid with the View Model pattern. But it's okay, it doesn't need to block this PR. We can just fix it as we continue working